rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc

```diff
@@ -92,74 +95,82 @@ Time zone for defining downtimes.
 Click *Create* to add a downtime to the application.
 . Click *Save*.
 
-Learn how to xref:myrpa-start.adoc#required-process-applications[assign required applications to a process].
-
-Learn how to xref:botmanagement-manage.adoc#bot-assign-servicetimes-applications[assign required applications to a bot].
-
 ==== Edit an Application
 
 Edit an application to change its data or down-times:
 
-. In the *Process Applications* view of the *Process Automation* module, click *Create*.
+. Open the *Process Applications* view of the *Process Automation* module.
+. Click *Create*.
 . Edit the form *Edit the application*.
++
+The properties are explained in the topic <<form-create-process-applictions, *Create an Application*>>.
 . Click *Save*.
 
 ==== Deactivate or Activate an Application
 
 Deactivate an application if it is temporarily not available and you don't know when it will be available again.
 
-Use down-times to schedule maintenance and down times. Deactivation periods are not added to the list of down-times.
+Use down-times to schedule maintenance and planned down times. Deactivation periods are not added to the list of down-times.
+
+Applications are not available during down-times or while deactivated. A member of the process team can deploy a process on a bot that has assigned the required processes. If an application becomes non-available, the bot completes running processes using that application and restarts them when the downtime or deactivation period has ended.
 
-Processes whose applications are currently down are executed to completion and are not restarted until the downtime or deactivation period has ended.
+If a application is not available, deactivate it:
```

Comment:
This sentence should read: "If **an** application is not available, deactivate it:"